### ISSUE TITLE
Problem with first cursor on rectangle region

### ISSUE THREAD

**Issue by jacobsun** [magnars/multiple-cursors.el#338]
![3](https://user-images.githubusercontent.com/1488997/41818586-8170e94c-77e4-11e8-943e-ce8db9d1e69e.gif)

The first cursor is not at the proper position when using `'rectangle-mark-mode` region,  is there a way to handle this?

**Comment by Fuco1** [comment]
What mc command are you calling? I suspect `mc/edit-lines`?  Is there any advantage to using `rectangle-mark-mode` here?

**Comment by DesiQuintans** [comment]
Yes, I also ran into this issue when I was trying to replicate RStudio/Ace Editor's implementation of multiple cursors, where users have the option of Alt-dragging to define a rectangular region with a cursor at every line on the left side of the rectangle. I can rectangle-select with the mouse just fine (<https://emacs.stackexchange.com/a/7261/19391>), but trying to run `mc/edit-lines` reproduces the above issue.

`rectangle-mark-mode` _can_ be used for this, but a multiple cursors approach is more flexible because I can just drag and type, rather than having to then call a function like `string-rectangle (C-t)` to replace text on every line inside the region.
